CONTRIBUTING.md

```diff
@@ -207,14 +207,23 @@ during development.
   to work with an AWS account that you have access to.  You can use the same user
   or role that you use for Github actions.
 
+#### run all tests
+
+```bash
+$ AWS_DEFAULT_REGION=eu-west-1 poetry run behave integration-tests/features --junit --junit-directory build/behave
+```
+
 _Note_: All integration tests are set up to run in `eu-west-*` region.  If you prefer
-to run in a different region you must update the region in each test before running it
+to run in a different region, you must update the region in each test before running it:
 
+```bash
+$ grep -r eu-west integration-tests
+```
 
-#### run all tests
+Edit the files.
 
 ```bash
-$ AWS_PROFILE=<profile> AWS_DEFAULT_REGION=<region> poetry run behave integration-tests/features --junit --junit-directory build/behave
+$ AWS_DEFAULT_REGION=<your-preferred-default-region> poetry run behave integration-tests/features --junit --junit-directory build/behave
```

Comment:
how about putting `AWS_PROFILE` back in?

Reply:
It seems overly prescriptive to me. I use this method to login to AWS:

```
export AWS_ACCESS_KEY_ID=your-access-key-id
export AWS_SECRET_ACCESS_KEY=your-secret-access-key
```

At least in the account where I run these integration tests. In my other accounts, I use the SSO method. 

Happy to put it back in if you really want it there.

Reply:
yes, i prefer it to be overly prescriptive.